streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -103,6 +103,7 @@
     public static final String VALUE_SERDE_CLASS_CONFIG = "value.serde";
     public static final String VALUE_SERDE_CLASS_DOC = "Serializer / deserializer class for value that implements the <code>Serde</code> interface.";
 
+    /**<code>user.endpoint</code> */
     public static final String USER_ENDPOINT_CONFIG = "user.endpoint";
     public static final String USER_ENDPOINT_DOC = "A user defined endpoint that can be used to connect to remote KafkaStreams instances. Should be in the format host:port";
```

Comment:
Question: Should we actually enforce "host:port"?  I have the feeling that the user should be allowed to put anything into this string.  For example, what about "http://server-1:1234" (= URI)?

The docstring should perhaps also say: "For example, an endpoint could be specified as host:port" (or sth like that).